Be.Vlaanderen.Basisregisters.AspNetCore.Mvc.Middleware.AddCorrelationIdToResponse.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why modify the .sln file?

Reply:
I reverted the sln, please squash the commits as I did not amend the original :)